packages/client/src/components/ClientApp.svelte

```diff
@@ -96,6 +96,9 @@
       <script src={`${plugin.jsUrl}`}></script>
     {/each}
   {/if}
+  <script
```

Comment:
Don't think I actually need this tag, I don't believe this menu is visible on the client.

Reply:
Yep this can be removed - it's builder only.